tutorials/data/combining with syncer example project/group 4/034d016f-a8b1-41c7-9099-86deec1540ae.json

```diff
@@ -0,0 +1,224 @@
+{
+  "name": "tutorials/data/Combining with syncer example project/Group 4",
```

Comment:
Should have more meaningful name.
Review the port and node names.
Review the data types.